Resources/Server Info/Guidebook/Chef/cookbook.xml

```diff
@@ -147,4 +147,16 @@ Protein can be used raw, but sometimes has to be cooked beforehand. Simply micro
 <GuideEntityEmbed Entity="FoodMeatSalami" Caption=""/>
 </Box>
 
+<Box Orientation="Vertical">
+	[color=#a4885c]Tofu[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 30u soy milk
+- 5u univerzal enzyme
```

Comment:
Denote the enzyme with (c) as a catalyst. If it's not actually a catalyst, make it one.

Reply:
Yep isn't a catalyst, making it be.